lib/prefix.c

```diff
@@ -1325,35 +1325,136 @@ static const char *prefixevpn2str(const struct prefix_evpn *p, char *str,
 	return str;
 }
 
+const char *num_str[256] = {
```

Comment:
This is buying performance at the cost of about a kilobyte of what is likely going to be L1 cache.  This is gonna be fast in synthetic benchmarking but application performance is a different question.

Do you have perf numbers with something that isn't an 1kB string table?

Reply:
(for the record, L1D$ on current mediocre ARM CPUs is frequently as low as 16kB)